src/Components/Tasks.tsx

```diff
@@ -1,25 +1,25 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 import Task from "./Task";
 import Swal from "sweetalert2";
-// import Fade from 'react-reveal/Fade';
-import Fade from 'react-reveal' ;
+
 import Modal from "./Modal";
 import TableHeader from './TableHeader';
 
-type Todo = {
+export type Todo = {
   id: number,
   name: string,
   assignee: string,
   startDate: string,
   endDate: string,
-  done: number
+  done: boolean
 
 }
 
 export default function Tasks() {
-  let [dataParsed, setDataParsed] = useState<Todo []>();
-  let [toggle, setToggle] = useState<number>(0);
+  let [dataParsed, setDataParsed] = useState<Todo []>([]);
+  let [toggle, setToggle] = useState<boolean>(false);
```

Comment:
`let [isToggled, setIsToggle] = useState<boolean>(false);`
we also don't have to  explicitly add boolean for state type since we are using false as the default value